etna/records/views/choosers.py

```diff
@@ -59,17 +58,29 @@ def get(self, request, pk):
             raise Http404
 
 
-class RecordChooserViewSet(ModelChooserViewSet):
+class RecordChooserViewSet(ChooserViewSet):
     """Custom chooser to allow users to filter and select records."""
 
     base_chosen_view_class = KongChosenView
     chooser_mixin_class = KongModelChooserMixinIn
     icon = "form"
-    model = RecordPage
+    model = Record
     page_title = "Choose a record"
     per_page = 10
-    order_by = "iaid"
-    fields = ["iaid", "title"]
+
+    def get_choose_view_attrs(self):
+        attrs = super().get_choose_view_attrs()
+        if hasattr(self, "model"):
+            attrs["model"] = self.model
+
+        return attrs
+
+    def get_chosen_view_attrs(self):
+        attrs = super().get_chosen_view_attrs()
+        if hasattr(self, "model"):
+            attrs["model"] = self.model
+
+        return attrs
```

Comment:
```suggestion
        attrs = super().get_chosen_view_attrs()
        attrs.update(model=self.model)
        return attrs
```

Reply:
Thanks both. Committed the change.

The unnecessary check was left behind from when we copied `ModelChooserViewSet.get_chosen_view_attrs` over to our chooser:

https://github.com/wagtail/wagtail-generic-chooser/blob/6d21c64d2f0e3ab6342def4548785c93df9be2db/generic_chooser/views.py#L670-L675